lib/pgerd/erd.rb

```diff
@@ -73,7 +74,17 @@ def create_a_digraph
     end
 
     def draw_the_tables
-      @database.tables.each { |table| draw_table(table) }
+      progress = ProgressBar.create({
+        :title => 'Tables',
+        :total => @database.tables.count,
+        :format => '%t: |%B| %%%p %E',
+	:output => $stderr
+	})
+
```

Comment:
I'd like to modernize the syntax:

```suggestion
      progress = ProgressBar.create(
        title: 'Tables',
        total: @database.tables.count,
        format: '%t: |%B| %%%p %E',
	output: $stderr)
```

By the way, how many tables do you have? For me, with tens of tables, this script runs in an instance.